person.rb

```diff
@@ -6,10 +6,11 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @id = Random.rand(1..1000)
     @name = name
     @age = age
+    @rentals = []
     @parent_permission = parent_permission
   end
 
-  attr_reader :id
+  attr_reader :id, :rentals
   attr_accessor :name, :age
 
```

Comment:
- [ ] As I mentioned in the previous comment, kindly make sure `Getters` and `Setters` are present for `@rentals` i.e `:rentals` has to be added on the `attr_accessor`. For more details, you can read **[this article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_principles_examples.md#useful-shortcuts-attr_reader-attr_writer-and-attr_accessor)**.

- [ **OPTIONAL** ] For **best practices** I think it would be better if you move the `attr_reader` and `attr_accessor` just before the constructor `def initialize(...)` method.